roxctl/scanner/uploaddb/uploaddb_test.go

```diff
@@ -42,7 +42,7 @@ func executeUpdateDbCommand(t *testing.T, serverURL string) (*bytes.Buffer, *byt
 
 	// We are using common.DoHTTPRequestAndCheck200 inside uploadDd(). This
 	// function uses  global variables that are set by command execution.
-	// TODO: Change uploadDd function to use HTTPClient from Environment.
+	// TODO: ROX-13638 Change uploadDd function to use HTTPClient from Environment.
```

Comment:
```suggestion
	// TODO(ROX-13638): Change uploadDd function to use HTTPClient from Environment.
```

This notation should prevent merging `ROX-13638` without addressing this TODO.